docs/src/content/docs/introduction/op-concepts.mdx

```diff
@@ -4,17 +4,17 @@ tableOfContents:
   maxHeadingLevel: 4
 ---
 
-import { LinkOut, CodeBlock } from '@interledger/docs-design-system'
+import { LinkOut, CodeBlock, Tooltip } from '@interledger/docs-design-system'
 
 The Open Payments standard helps to facilitate the setup of payments between Open Payments-enabled payment accounts.
 
-When setting up a payment, the client must obtain the wallet address for both the payer and payee. Since the payer is typically the client's end-user, the client can obtain the payer's wallet address during an onboarding process, for example. The payer must be authenticated by their account servicing entity (ASE) to grant the client the permission it needs to access the their Open Payments-enabled account.
+When setting up a payment, the <Tooltip content="A mobile or web application or service" client:load>client</Tooltip> must obtain the wallet address for both the sender and recipient. Since the sender is typically the client's user, the client can obtain the sender's wallet address during an onboarding process, for example. The sender must be authenticated by their account servicing entity (ASE) to grant the client the permission it needs to access the their Open Payments-enabled account.
```

Comment:
Tooltip content="**Such as** a mobile or web application or service"

We should be clear that those are examples.